### PR TITLE
Remove references to requirements-dev.txt

### LINKED ISSUE #654: Remove remaining references to requirements-dev.txt
There are still a few references to the now-removed requirements-dev.txt file.

### PR DESCRIPTION
**Pull request by CruiseDevice**
Fixes #654

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Looks good but I will give @palnabarun the final pass.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ It is advised to install all the requirements inside [virtualenv], use [virtuale
 sudo apt-get update
 sudo apt-get upgrade
 sudo apt-get install libpq-dev python-dev build-essential tcl
-pip install -r requirements-dev.txt
+pip install -r requirements.txt
```
Comment (palnabarun) [inline README.md:24]: The README has changed and does not contain this block anymore.

**Comment by palnabarun** [comment]
@CruiseDevice -- Thank you for your contribution. :+1: 

Please fix the conflicts.

> Note: I edited the PR comment to reflect that merging this with fix #654

**Comment by palnabarun** [comment]
@CruiseDevice -- can you please rebase this branch on top of master instead of merging master to this branch? We avoid merging merge commits.

**Comment by ananyo2012** [comment]
@palnabarun What is the use of `bin/regenerate.sh` ? Do we need this anymore ?

**Comment by palnabarun** [comment]
@ananyo2012 -- No idea. But, I prefet keeping it for now for sanity.

**Pull request merged**